.env_config_sample

```diff
@@ -1,2 +1,2 @@
-export DATABASE_URI='postgres://USER:PASSWORD@HOST:PORT/NAME'
-export SECRET_KEY='AuthorsHaven'
+export DATABASE_URI='postgres://postgres:password@localhost:5432/authors_haven'
+export SECRET_KEY='AuthorsHaven
```

Comment:
@ezrogha kindly replace the quotation mark and leave a blank line at the bottom.

Reply:
Thanks @dannylwe